core/src/banking_stage.rs

```diff
@@ -884,6 +1100,9 @@ impl BankingStage {
         tracer_packet_stats: &mut TracerPacketStats,
         bank_forks: &Arc<RwLock<BankForks>>,
     ) {
+        if unprocessed_transaction_storage.should_not_process() {
+            return;
+        }
```

Comment:
To clarify, banking stage still has a thread to receive and buffer gossip votes, but not processing them at all?

Reply:
read further, so gossip and tpu votes are received in separate threads, but are stored at same  Latest_unprocessed_votes. So effective, gossip vote threads just do receiving and storing; TPU vote thread processes all vote transactions (on top of receiving tpu votes). Hope I got this right.